commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -51,7 +51,7 @@ public class Eth2SubCommand extends ModeSubCommand {
       names = {"--network"},
       paramLabel = "<NETWORK>",
       description =
-          "Predefined network configuration to use. Possible values: [mainnet, pyrmont, prater, kintsugi, kiln, gnosis, minimal], file path"
+          "Predefined network configuration to use. Possible values: [mainnet, pyrmont, prater, kintsugi, kiln, gnosis, minimal, ropsten], file path"
```

Comment:
Could also remove kintsugi to keep in line with teku: https://github.com/ConsenSys/teku/commit/b329dc05bce59c3cc484f260013ac584d71ebe40

Reply:
Another reason to use the enum somehow